improver_tests/spotdata/test_SpotLapseRateAdjust.py

```diff
@@ -312,6 +312,15 @@ def test_different_dimensions(self):
         for slice in result.data:
             self.assertArrayEqual(slice, expected)
 
+    def test_using_fixed_lapse_rates(self):
+        """Test that the data is as expected when using fixed lapse rates."""
+
+        for lr in [0.5 * DALR, DALR]:
```

Comment:
Can you add a zero value? My understanding is that the code should still run, but the data should not change.

Reply:
Added.